### PR TITLE
fix(perks-and-openings): fixes svg issue and responsiveness

### PR DESCRIPTION
**Pull request by rajvaya**
fixes svg color issue
adds responsiveness to openings cards
adds Openings Email CTA

### CODE REVIEW

File: src/containers/perks-and-benefits/perks-and-benefits.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react'
```
Comment (pallavisaraogi) [inline src/containers/perks-and-benefits/perks-and-benefits.tsx:1]: why separate folders for each file?

File: src/components/openingsCTA.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```
Comment (ulhas) [inline src/components/openingsCTA.tsx:1]: we can club these two imports together

File: src/components/positionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+export interface PositionCardProps {
+  title: string
+  tags: string[]
+  url: string
+}
+
+const PositionCard: FC<PositionCardProps> = ({ title, tags, url }) => {
+  return (
+    <a
```
Comment (ulhas) [inline src/components/positionCard.tsx:14]: Gatsby suggests we `Link` for pages and `a href` for external stuff.

File: src/components/toggleTheme.tsx [hunk 4]
```diff
@@ -1,17 +1,22 @@
-import React from 'react'
+import React, { BaseSyntheticEvent } from 'react'
 import { ThemeToggler } from 'gatsby-plugin-dark-mode'
+import Sun from '../assets/images/light-sun.svg'
+import Moon from '../assets/images/dark-moon.svg'
 
-import { Moon, Sun } from '../../images'
+interface ThemeToggleProps {
+  theme: string
+  toggleTheme: (e: string) => void
```
Comment (ulhas) [inline src/components/toggleTheme.tsx:8]: can we change `e` to a more meaningful name? i guess it's takes in the preference for theme, right?

File: src/containers/openings/openings.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import PositionCard, { PositionCardProps } from '../../components/positionCard'
+import OpeningsCTA from '../../components/openingsCTA'
+
+interface OpeningsProps {
+  openings: PositionCardProps[]
+}
+
+const Openings: FC<OpeningsProps> = ({ openings }) => {
+  return (
+    <div className="mx-8 py-10 lg:pb-11 lg:mx-32 lg:pt-32">
```
Comment (ulhas) [inline src/containers/openings/openings.tsx:11]: `margin` classes should be applied externally via `className` prop

Comment (ulhas) [reply src/containers/openings/openings.tsx:11]: for the root div in a component. having margin inside the component makes it non resuable.

**Review by ulhas — CHANGES_REQUESTED** [review]

File: src/containers/openings/openings.tsx (hunk 5 above)
Comment (rajvaya) [reply src/containers/openings/openings.tsx:11]: already its in root div openings.tsx is root container

**Review by ulhas — APPROVED** [review]

**Pull request merged**